marshmallow_jsonschema/base.py

```diff
@@ -268,7 +269,7 @@ def _get_schema_for_field(self, obj, field):
             if isinstance(field, fields.Nested):
                 # Special treatment for nested fields.
                 schema = self._from_nested_schema(obj, field)
-            elif ALLOW_UNIONS and isinstance(field, Union):
```

Comment:
Could you cleanup any other instances of `ALLOW_UNIONS` in the repo. I think we can get away without importing any of the union libraries now.